packages/perseus/src/widgets/radio/radio-validator.ts

```diff
@@ -21,23 +23,33 @@ function radioValidator(
         };
     }
 
-    // TODO(LEMS-2541): should numCorrect actually be on the rubric
-    // instead of the userInput?
-    if (
-        userInput.numCorrect &&
-        userInput.numCorrect > 1 &&
-        numSelected !== userInput.numCorrect
-    ) {
+    const numCorrect: number = _.reduce(
+        rubric.choices,
+        function (memo, choice) {
+            return choice.correct ? memo + 1 : memo;
+        },
+        0,
+    );
```

Comment:
When it's an option, I'd like to move away from underscore and use native JS; [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) is native now.

```suggestion
    const numCorrect: number = rubric.choices.reduce(
        (acc, curr) => {
            return curr.correct ? acc + 1 : acc;
        },
        0,
    );
```

Reply:
I also try to get rid of underscore stuff. Sometimes my brain refuses to realize I can update something I copy and pasted from another part of the codebase 😂 Thanks! Updated~